lib/registries/addon/overview/route.ts

```diff
@@ -27,21 +28,34 @@ export default class Overview extends GuidRoute {
         const registration: Registration = yield model.taskInstance;
 
         if (registration) {
-            const contributors: SparseModel[] = yield registration.sparseLoadAll(
-                'bibliographicContributors',
-                {
-                    contributor: ['users', 'index'],
-                    user: ['fullName'],
-                },
-            );
-
-            const institutions: SparseModel[] = yield registration.sparseLoadAll(
-                'affiliatedInstitutions',
-                { institution: ['name'] },
-            );
-
-            const license = yield registration.license;
-
+            let contributors: SparseModel[] = [];
+            let institutions: SparseModel[] = [];
+            let license;
+            let identifiers: Identifier[] = [];
+
+            try {
+                [
+                    { value: contributors },
+                    { value: institutions },
+                    { value: license },
+                    { value: identifiers },
+                ] = yield allSettled([
+                    registration.sparseLoadAll(
+                        'bibliographicContributors',
+                        { contributor: ['users', 'index'], user: ['fullName'] },
+                    ),
+                    registration.sparseLoadAll(
+                        'affiliatedInstitutions',
+                        { institution: ['name'] },
+                    ),
+                    registration.license,
+                    registration.identifiers,
+                ]);
+            } catch (e) {
+                // Ignore rejected promises;
```

Comment:
I like the `allSettled` refactor, but ignoring all exceptions is something we should always avoid.

This change was because `affiliatedInstitutions` isn't exposed for withdrawn registrations, right? If you wouldn't mind a quick backend PR changing [the `ExcludeWithdrawals`permission](https://github.com/CenterForOpenScience/osf.io/blob/fb7c711eabc09795f8d1741677651e977dc10449/api/nodes/views.py#L1443) to `ReadonlyIfWithdrawn` then we could avoid that problem.

Reply:
As opposed to `registrations.identifiers`, we don't show affiliated institutions info on the tombstone page, probably for a good reason? so making `registration.affiliatedInstitutions` `readOnlyIfWithdrawn` would solely be for head meta tags.